JavaErickHernandez/Day3/Lists.java

```diff
@@ -0,0 +1,116 @@
+package bootcamp;
+import java.util.*;;
+
+public class Lists {
+	static List<Integer> numbers = new ArrayList<>();
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableList, nameSet, etc...